### DISCUSSION TITLE
Chia Will Not Start

### DISCUSSION THREAD

**Discussion by FPD315** [Chia-Network/chia-blockchain#18285] [Support]
I down loaded the latest version of the Chia program and now it will not work.  When I start the Chia program it will start loading the services and then get stuck on  "Starting Services" "Full Node".  What can I do to fix this issue?

**Comment by BrandtH22** [comment]
Hey @FPD315 , this is generally related to 1 of 2 things. Either your system needs to have its blockchain database updated from v1 to v2 or your system has more than 61 cores and needs another config setting to be updated.

NOTE - make sure to fully stop chia before making any changes to the needed files.

To verify if the blockchain db needs to be updated navigate to `~/.chia/mainnet/db/` and look at the .sqlite file, if it has V1 in the title then we need to update and you can proceed with the steps outlined here:
https://docs.chia.net/faq/#how-do-i-upgrade-my-version-1-blockchain-database-to-version-2

If this is not the case then I would look at your log file located at `~/.chia/mainnet/log/debug.log` for a line that indicates ValueError: max_workers must be <= 61 . If this line is present then you will need to navigate to the config file at `~/.chia/mainnet/config/config.yaml `and in the full_node section update the parameter reserved_cores to 10.

If neither of these are the issue let us know and we can further debug.

As a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia)

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
